### PR TITLE
Hello word stateless

### PR DESCRIPTION
**Pull request by EEtancelin**
Correcting https://github.com/shakacode/react_on_rails/issues/653

Add a Hello word component for the redux generator.
Add a mapDispatchToProps function in the container.
Update the template generator

---
This change is [Reviewable](https://reviewable.io/reviews/shakacode/react_on_rails/659)

### CODE REVIEW

**Comment by EEtancelin** [comment]
Hello Justin, The build fail, i'm actually readinf

**Comment by justin808** [comment]
@EEtancelin why did you close this?

**Comment by EEtancelin** [comment]
I do it by mystake, i don't even reminder closing it.

**Comment by justin808** [comment]
@EEtancelin Please see if you can get CI to pass.

File: lib/generators/react_on_rails/templates/redux/base/client/app/bundles/HelloWorld/components/HelloWord.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React, { PropTypes } from 'react';
```
Comment (justin808) [inline lib/generators/react_on_rails/templates/redux/base/client/app/bundles/HelloWorld/components/HelloWord.jsx:1]: Incorrect file name "Word" ==> "World"

Comment (justin808) [reply lib/generators/react_on_rails/templates/redux/base/client/app/bundles/HelloWorld/components/HelloWord.jsx:1]: be sure to run linting as well.

**Comment by justin808** [comment]
@EEtancelin I'd like to get this wrapped up today.Otherwise, I might take this on.

**Comment by justin808** [comment]
I put some notes in this gist: https://gist.github.com/justin808/c73089d97680c09cc8e032c5317abb3c

**Comment by justin808** [comment]
See #669

**Pull request closed without merging**